alerts_consumer.go

```diff
@@ -118,10 +118,12 @@ func (c *AlertsConsumer) encodeMessage(fields map[string]interface{}) ([]byte, [
 				switch t := dimVal.(type) {
 				case string:
 					dims[dim] = t
-					cwDims = append(cwDims, &cloudwatch.Dimension{
-						Name:  aws.String(dim),
-						Value: &t,
-					})
+					if dim != "Hostname" && dim != "env" {
```

Comment:
could reference `defaultDimensions` here and/or change the name of that variable to `sfxDefaultDimensions`